src/fixed-footer-layout.tsx

```diff
@@ -4,10 +4,13 @@ import debounce from 'lodash/debounce';
 import {createUseStyles} from './jss';
 import {isRunningAcceptanceTest} from './utils/platform';
 import {useElementDimensions, useIsomorphicLayoutEffect, useScreenSize, useTheme} from './hooks';
-import {Portal} from './portal';
-import {addPassiveEventListener, removePassiveEventListener} from './utils/dom';
+import {addPassiveEventListener, getScrollableParentElement, removePassiveEventListener} from './utils/dom';
 
-const getScrollDistanceToBottom = () => document.body.scrollHeight - window.innerHeight - window.scrollY;
+const getScrollDistanceToBottom = (el: HTMLElement) => el.scrollHeight - el.scrollTop - el.clientHeight;
+
+const getScrollEventTarget = (el: HTMLElement) => (el === document.documentElement ? window : el);
+
+const hasScroll = (el: HTMLElement) => el.scrollHeight > el.clientHeight;
```

Comment:
could these functions be utility functions in dom.tsx?

Reply:
sure! moved `getScrollDistanceToBottom` and `hasScroll`